routes/keyword.js

```diff
@@ -7,19 +7,41 @@ const Keyword = require('../models/KeywordModel.js')
 
 const router = express.Router()
 
-router.get('/:word', (req, res) => {
-    console.log(req.params)
+router.get('/word/:word', (req, res) => {
     let word = req.params.word
-    console.log(word)
     Keyword.find({word: word}).lean().then((wordList) => {
-        console.log(wordList[0])
-        console.log(wordList[0].votes)
-        console.log(word)
         res.send(wordList[0].votes)
     }).catch((err) => {
-        console.log(err)
         res.status(500).end()
     })
 })
 
+router.get('/top', (req, res) => {
+	Keyword
+		.find()
+		.lean()
+		.then((wordList) => {

```

Comment:
This method is fine for this PR, but I think you should change this to use `.aggerate` in the future, so you don't load all of the keyword documents in memory to do the sorting

More on `.aggerate`:  http://mongoosejs.com/docs/api.html#aggregate-js
An example: http://stackoverflow.com/questions/26623141/mongoose-aggregate-and-sort